extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -1614,59 +1648,58 @@ export class AmpA4A extends AMP.BaseElement {
   }
 
   /**
+   * @param {!./head-validation.ValidatedHeadDef} headData
    * @param {function()} checkStillCurrent
    * @return {!Promise} Whether the creative was successfully rendered.
    */
-  renderFriendlyTrustless_(checkStillCurrent) {
-    return this.sanitizedHeadPromise_.then((headData) => {
-      checkStillCurrent();
-      devAssert(this.element.ownerDocument);
-      this.maybeTriggerAnalyticsEvent_('renderFriendlyStart');
-
-      const {height, width} = this.creativeSize_;
-      const {extensions, fonts, head} = headData;
-      this.iframe = createSecureFrame(
-        this.element.ownerDocument,
-        head,
-        this.getIframeTitle(),
-        height,
-        width
-      );
-      this.applyFillContent(this.iframe);
+  renderFriendlyTrustless_(headData, checkStillCurrent) {
+    checkStillCurrent();
+    devAssert(this.element.ownerDocument);
+    this.maybeTriggerAnalyticsEvent_('renderFriendlyStart');
+
+    const {height, width} = this.creativeSize_;
+    const {extensions, fonts, head} = headData;
+    this.iframe = createSecureFrame(
+      this.element.ownerDocument,
+      head,
+      this.getIframeTitle(),
+      height,
+      width
+    );
+    this.applyFillContent(this.iframe);
 
-      // TODO(ccordry): FIE does not handle extension versioning, but we have
-      // it accessible here.
-      const extensionIds = extensions.map((extension) => extension.extensionId);
+    // TODO(ccordry): FIE does not handle extension versioning, but we have
```

Comment:
Not a subject to this pull request, but I don't think there's any reason left to not support versions. They just need to be passed through - that's it. Could you pls schedule a follow up on this?

Reply:
Sure, created #29604